source/chapters/appendix/mixxx_controls.rst

```diff
@@ -820,6 +820,19 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 2.0.0
 
 
+.. mixxx:control:: [ChannelN],bpm_toggle_lock
+                   [PreviewDeckN],bpm_toggle_lock
+                   [SamplerN],bpm_toggle_lock
+
+   Toggle the :term:`beatgrid`/:term:`BPM` lock state.
```

Comment:
Just a comment:
I think 'beatgrid' (and some other terms like phase and tempo) should be translated
```
:term:`beatgrid <beatgrid>`
```
but it is used like this elsewhere, so ignore...

Reply:
I'm a regex master, do you want me to give it a go?

Reply:
Replacing `:term:\`(\w+)\`` with `\`:term:\`$1 <$1>` would do the trick, wouldn't it?   
(I'm on VSCode so `$` instead of `\`)

Reply:
Nope, thanks RegEx master : )
Before doing a mass-replace let's discuss that on Zulip first.